ci/lint/06_script.sh

```diff
@@ -21,6 +21,7 @@ test/lint/lint-all.sh
 
 if [ "$TRAVIS_REPO_SLUG" = "bitcoin/bitcoin" ] && [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
     git log --merges --before="2 days ago" -1 --format='%H' > ./contrib/verify-commits/trusted-sha512-root-commit
-    travis_retry gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys $(<contrib/verify-commits/trusted-keys) &&
+    ${CI_RETRY_EXE} gpg --keyserver hkps://keys.openpgp.org    --recv-keys $(<contrib/verify-commits/trusted-keys) &&
+    ${CI_RETRY_EXE} gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys $(<contrib/verify-commits/trusted-keys) &&
```

Comment:
Why not put the key servers into an array and loop through it until it finds keys. Maybe it's a bit too much for this PR but if we add more backup servers from time to time it'll get useful